### DISCUSSION TITLE
Remove the glowing effect from potions

### DISCUSSION THREAD

**Discussion by Krutoy242** [ACGaming/UniversalTweaks#479] [Tweak Suggestions]
Please add `RandomPatches` feature to UT that removing potion glint effect - its very handy with modpacks with tonns of potions since its easier to determine potion.

Config defined in RP:
```ruby
    # Whether to remove the glowing effect from potions.
    # Default: false
    B:removePotionGlint=true
```

How it looks in game:

![image](https://github.com/ACGaming/UniversalTweaks/assets/936618/b870d3b9-d135-4719-bb5d-87e3b7f509b3)

**Comment by ACGaming** [comment]
Implemented in 76abfcd7f2bc479625d2b1d3386a26b0413b6c2f
